setup.py

```diff
@@ -87,6 +90,58 @@ def download_file(URL, dest_dir, local_file, num_retries=4):
     return return_code
 
 
+def app_available(app_name):
```

Comment:
This function doesn't follow the numpy standard. Also, has been it "copied" from Qiime? If so, you will need to get explicit permission from the devs in QIIME, so we can change the license here.